src/BuildOpt.h

```diff
@@ -7,21 +7,16 @@
   #define RADIOLIB_BUILD_ARDUINO
 #else
   // generic build
+  #include <stdio.h>
```

Comment:
Why is this needed?

Reply:
Similar with other includes (ctypes, math.h etc.) in other files.

Reply:
There indeed useless for Arduino build since they are already included, but on generic build they are required to use some functions like `sprinf`, `max`, `min`, `fabs`, etc.
These includes did not cause any problems with arduino, so I left them unconditioned at the top of the file using functions from them.

Reply:
Thanks for the examplanation, in that case this can remain.